### PR TITLE
query-generation

### PR DESCRIPTION
**Pull request by AnjaliWani96**
Query Generation view and utils added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "antd": "^4.0.3",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "react-scripts": "3.4.0"
+    "react-scripts": "^3.4.0",
+    "yarn": "^1.22.4"
```
Comment (balgovindy) [inline package.json]: why you add yarn here?

File: src/component/QueryGenerator/index.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react'
+// import ReactDOM from "react-dom";
+import "antd/dist/antd.css";
```
Comment (balgovindy) [inline src/component/QueryGenerator/index.js]: this line should be removed

File: package.json (hunk 1 above)
Comment (AnjaliWani96) [reply package.json]: i had installed yarn when i was facing errors with yarn start.
Now i have uninstalled

File: src/component/QueryGenerator/index.js (hunk 2 above)
Comment (AnjaliWani96) [reply src/component/QueryGenerator/index.js]: okay.

**Pull request merged**